test/apitests.js

```diff
@@ -30,7 +30,7 @@ describe("GET /todo-item/", () => {
         res.should.have.status(200);
 
         res.body.should.be.an('array');
-        res.body.should.have.length(2);
+        res.body.should.have.length(0);
```

Comment:
```suggestion
        res.body.should.have.length(2);
```

please revert back to 2